index.Rmd

```diff
@@ -83,15 +83,15 @@ summary(fit)
 Show tables, plots, etc. and describe them.
 
 ```{r echo=TRUE, warning=FALSE}
-ggplotly (g, tooltip = c("County", "Opioid.Poisoning.Deaths", "Year"))
+ggplotly (g, tooltip = c("County", "Opioid.Poisoning.Deaths", "Year")) 
     
```

Comment:
The interactivity is convenient and user-friendly. I like it! But the title overlays with the legend title, which affect the readability.